src/components/clusterDetail/ClusterDetail.tsx

```diff
@@ -30,35 +30,24 @@ import { AlertsContext } from '../AlertsContextProvider';
 import { canDownloadClusterLogs } from '../hosts/utils';
 
 const canAbortInstallation = (cluster: Cluster) => {
-  if (
-    !['preparing-for-installation', 'installing', 'installing-in-progress'].includes(cluster.status)
-  ) {
-    return false;
-  }
-
-  if (cluster.hosts) {
-    if (cluster.hosts.find((h) => h.status === 'installing-pending-user-action')) {
-      // a host in installing-pending-user-action
-      return false;
-    }
-
-    if (cluster.hosts.find((h) => h.status === 'error')) {
-      // a host is in error
-      if (cluster.hosts.find((h) => h.status !== 'installing')) {
-        // no host is in installing
-        if (
-          cluster.hosts.find(
-            (h) => !['error', 'installed', 'preparing-for-installation'].includes(h.status),
-          )
-        ) {
-          // a host not yet in error or installed or preparing-for-installation
-          return false;
-        }
-      }
-    }
-  }
-
-  return true;
+  const allowedClusterStates = [
+    'preparing-for-installation',
+    'installing',
+    'installing-in-progress',
+  ];
+  const allowedHostStates = [
+    'preparing-for-installation',
+    'pending-user-action',
+    'installing',
+    'installing-in-progress',
+    'installed',
+    'error',
+  ];
+  const { hosts = [] } = cluster;
+  return (
+    allowedClusterStates.includes(cluster.status) &&
+    hosts.find((host) => !allowedHostStates.includes(host.status))
```

Comment:
According to this discussion it seems that it is not even necessary to check the host status https://coreos.slack.com/archives/CUPJTHQ5P/p1603284140247300?thread_ts=1603278706.226100&cid=CUPJTHQ5P